SYCL/ESIMD/api/functional/ctors/ctor_load_acc_core.cpp

```diff
@@ -0,0 +1,76 @@
+//==------- ctor_load_acc_core.cpp  - DPC++ ESIMD on-device test -------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu, level_zero
+// XREQUIRES: gpu
+// TODO gpu and level_zero in REQUIRES due to only this platforms supported yet.
+// The current "REQUIRES" should be replaced with "gpu" only as mentioned in
+// "XREQUIRES".
+// UNSUPPORTED: cuda, hip
+// RUN: %clangxx -fsycl %s -fsycl-device-code-split=per_kernel -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+//
+// Test for simd load from accessor constructor.
+// The test uses reference data and different alignment flags. Invokes simd
+// constructors in different contexts with provided reference data and alignment
+// flag using accessor as input.
+// It is expected for destination simd instance to store a bitwise same data as
+// the reference one.
+
+#include "ctor_load_acc.hpp"
+#include "ctor_load_acc_coverage.hpp"
+
+using namespace esimd_test::api::functional;
+
+int main(int, char **) {
+  sycl::queue queue(esimd_test::ESIMDSelector{},
+                    esimd_test::createExceptionHandler());
+
+  bool passed = true;
+
+  // Define input for simd<T,N> combinations
+  const auto types = get_tested_types<tested_types::core>();
+  const auto sizes = get_all_sizes();
+
+  // Define input for sycl::accessor<T,Dims,Mode,Target> type combinations
+  const auto acc_dims = ctors::coverage::get_all_dimensions();
+  const auto acc_modes = ctors::coverage::get_all_modes();
+  const auto acc_targets = ctors::coverage::get_all_targets();
+
+  // Define input for constructor calls
+  // Offset generator is used to have offset to be multiplier of alignment flag
+  const auto contexts = ctors::coverage::get_all_contexts();
+  const auto offset_generators = ctors::coverage::get_all_offset_generators();
+
+  // Run over all types with element and vector alignment
+  // Default tag for overaligned is configured in a way appropriate for any
+  // type from the type coverage, so run over it as well
+  {
+    const auto alignments =
+        named_type_pack<ctors::alignment::element, ctors::alignment::vector,
```

Comment:
I think oword should be one of the alignment cases as well. Is overal guaranteed to be a multiple of oword?

Reply:
It's equal to `oword`. Though maybe it's better to remove default parameter at all and make alignment used explicit.

It would be a tiny fix just moving code around, so I'll make it in a separate PR today to ensure you could take a look at the suggestion without blocking this PR's merge. Thanks for the catch.